q2_types/_feature_table.py

```diff
@@ -97,6 +97,9 @@ def feature_table_to_pandas_dataframe(data_dir):
 plugin.register_semantic_type(RelativeFrequency)
 plugin.register_semantic_type(PresenceAbsence)
 
-plugin.register_type_to_data_layout(
-    FeatureTable[Frequency | RelativeFrequency | PresenceAbsence],
```

Comment:
I don't remember the specifics of how this registration worked, but did it use a `dict` for reverse lookup? If so then :+1:  as identity/hashing isn't semantic, but if not I think the single registration is better...